config/config.php

```diff
@@ -48,6 +48,12 @@
  */
 $GLOBALS['FE_MOD']['user']['lostPasswordNotificationCenter'] = 'ModulePasswordNotificationCenter';
 
+if (isset($GLOBALS['FE_MOD']['newsletter'])) {
```

Comment:
I think you should use `ModuleLoader::isActive('newsletter')` here

Reply:
Changed in 9678fd6.